power_grid_model_c/power_grid_model/include/power_grid_model/common/three_phase_tensor.hpp

```diff
@@ -284,6 +284,16 @@ inline std::pair<DoubleComplex, DoubleComplex> inv_sym_param(DoubleComplex const
     return {(s + m) * det_1, -m * det_1};
 }
 
+// inverse of symmetric tensor
+template <symmetry_tag sym> inline ComplexTensor<sym> inv_sym_tensor(ComplexTensor<sym> const& x) {
+    if constexpr (is_symmetric_v<sym>) {
+        return 1.0 / x;
+    } else {
+        auto [s, m] = inv_sym_param(x(0), x(1));
+        return ComplexTensor<sym>(s, m);
+    }
+}
+
```

Comment:
This function is never used. Maybe delete it?

Reply:
You are right. It was being used in a previous implementation, but not anymore. I will remove it.